tests/integration/schema/schema.spec.ts

```diff
@@ -1,20 +1,28 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 import httpStatusCodes from 'http-status-codes';
+import { Redis } from 'ioredis';
 import { container } from 'tsyringe';
+import { redisSymbol } from '../../../src/common/constants';
 import { Schema } from '../../../src/schema/models/types';
 import { Tags } from '../../../src/schema/providers/fileProvider/fileProvider';
 
 import { registerTestValues } from '../testContainerConfig';
 import * as requestSender from './helpers/requestSender';
 
 describe('schemas', function () {
+  let redisConnection: Redis;
   beforeAll(async function () {
     await registerTestValues();
     requestSender.init();
+    redisConnection = container.resolve<Redis>(redisSymbol);
+    await redisConnection.flushall();
   });
   afterEach(function () {
     container.clearInstances();
   });
```

Comment:
this is not needed. you only resolve stuff from the container in beforeAll, not in each test

Reply:
done